react-accordion/my-app/src/TopicCard.tsx

```diff
@@ -0,0 +1,29 @@
+type Props = {
+  title: string;
+  content: string;
+  currentTopic: string;
+  onTitleClick: (str: string) => void;
+};
+
+export function TopicCard({
+  title,
+  content,
+  currentTopic,
+  onTitleClick,
+}: Props) {
+  return (
+    <>
+      <h2
+        onClick={() => onTitleClick(title)}
+        className="my-[-1px] border-2 border-black bg-[#ddd]">
+        {title}
+      </h2>
+      <p
+        className={`my-[-1px] border-2 border-black ${
+          currentTopic === title ? '' : 'hidden'
```

Comment:
Since you aren't doing any animations, I think this is easier with conditional rendering instead of using 'hidden'.